apps/backend/src/scripts/parsetdi17.ts

```diff
@@ -0,0 +1,16 @@
+import { handler } from '../lambdas/parseFlatFile';
+
+const formatOutput = (filepath: any) => {
+  const date = filepath.split('').splice(0, 10).join('');
+  filepath = filepath?.split('.').shift();
```

Comment:
(I know this looks ugly. I wanted to check on the folder/naming structure, and also this is temp...  )

Reply:
I don't think we need to prefix the date here. the file types could be the high level folder and all the files could go into the same folder.  

Probably a year folder under each type is good (even month maybe?? but We don't want to deal the logic for cross month navigation, so I am happy with just a year folder

Reply:
/<S3_BUCKET>/garms/2022/sales/<365 ish json files>

/<S3_BUCKET>/garms/2022/tdi17/raw/<365 ish dat files>
/<S3_BUCKET>/garms/2022/tdi17/parsed/<365 ish json files>

/<S3_BUCKET>/garms/2022/tdi34/raw/<365 ish dat files>
/<S3_BUCKET>/garms/2022/tdi34/parsed/<365 ish json files>

/<S3_BUCKET>/garms/2022/ddf/raw/<365 ish dat files>
/<S3_BUCKET>/garms/2022/ddf/parsed/<365 ish json files>


same for labor and any new ministry? 

we don't have to change this *NOW*, I am okay with logging this in the backlog